element_deeplabcut/model.py

```diff
@@ -748,7 +781,18 @@ def make(self, key):
                 output_directory=output_dir,
             )
             def do_analyze_videos():
-                from deeplabcut.pose_estimation_tensorflow import analyze_videos
+                engine = dlc_model_["engine"]
+                if engine is None:
+                    logger.warning(
+                        "DLC engine not specified in config file. Defaulting to TensorFlow."
+                    )
+                    engine = "tensorflow"

```

Comment:
This `if` is no longer needed, right?

Reply:
I kept this in case someone did not use the class method to insert into Model for whatever reason and inserted a blank string (which would technically still be allowed with the way the table is configured).